release.sh

```diff
@@ -35,76 +17,31 @@ if $aws_disabled; then
     exit 0
 fi
 
-# Commit info
-COMMIT="${YB_GIT_COMMIT:-}"
-if [ -z "${COMMIT}" ]; then
-    # If git is installed
-    if hash git; then
-        COMMIT="$(git rev-parse HEAD)"
-    fi
-fi
-
-BUILD_COMMIT_INFO=""
-if [ -n "${COMMIT}" ]; then
-    BUILD_COMMIT_INFO=" -X 'main.commitSHA=$COMMIT'"
-fi
-
-umask 077
-
 echo "Releasing ${CHANNEL} yb version ${VERSION} [${COMMIT}]..."
 
-set -x
-
-rm -rf release
-mkdir -p release
-
-OSLIST=( linux darwin )
-ARCHLIST=( amd64 )
-
-for os in "${OSLIST[@]}"
-do
-  for arch in "${ARCHLIST[@]}"
-  do
-    GOOS=${os} GOARCH=${arch} go build -ldflags "-X 'main.version=$VERSION' -X 'main.date=$(date -u '+%F-%T')' -X 'main.channel=$CHANNEL'${BUILD_COMMIT_INFO} -s -w" -o release/yb-${os}-${arch}
-  done
-done
-
 (
-  cd release
-
-  for i in *
-  do
-    tar zcvf $i-${VERSION}.tgz $i
-    rm $i
-  done
-
-    for i in *.tgz
-    do
-      if [ -z "${local_test_release}" ]; then
-        if aws s3 ls s3://yourbase-artifacts/yb/${VERSION}/$i; then
-            echo "A version for ${VERSION} already exists! Not releasing this version."
-            exit 1
-        fi
-
-        aws s3 cp $i s3://yourbase-artifacts/yb/${VERSION}/
-      else
-        echo "Local test, would run:"
-        echo aws s3 ls s3://yourbase-artifacts/yb/${VERSION}/$i
-        echo aws s3 cp $i s3://yourbase-artifacts/yb/${VERSION}/
-      fi
+    for r in yb-*-*-${VERSION}.*; do
+        for bucket in "yourbase-artifacts/yb/${VERSION}/ yourbase-cats-bundles/"; do
```

Comment:
Missing quotes (will only loop once, this is a logic error):

```suggestion
        for bucket in "yourbase-artifacts/yb/${VERSION}/" "yourbase-cats-bundles/"; do
```